src/picker/VuePersianDatetimePicker.vue

```diff
@@ -1582,14 +1588,16 @@ export default {
       return date.clone()
     },
     convertToLocaleNumber(value) {
-      if (this.locale == 'fa') {
-        return `${value}`.replace(/\d+/g, function(digit) {
-          var ret = ''
-          for (var i = 0, len = digit.length; i < len; i++) {
-            ret += String.fromCharCode(digit.charCodeAt(i) + 1728)
-          }
-          return ret
-        })
+      if (this.convertNumbers) {
```

Comment:
```javascript
if (this.convertNumbers) {	      
 if (this.locale == 'fa') {
   //....
  }
}
```
can be replaced with `if (this.convertNumbers && this.locale === 'fa')`

Reply:
How about other locales than 'fa' ?

Reply:
I did not say anything about it, you added this condition yourself.

Reply:
localization support is added in the last version and the condition is because of that. As you know numbers in other locales like 'ar' looks different.